### ISSUE TITLE
AllNightfall DestinyActivityModeType?

### ISSUE THREAD

**Issue by t3rr11** [Bungie-net/api#747]
Any chance in the future could we get an AllNightfall DestinyActivityModeType? Since there are 4 modetypes for the nightfall (nightfall, heroicNightfall, scored_nightfall, scored_heroicNightfall) mode i thought this could be a handy addition.

**Comment by vthornheart-bng** [comment]
I agree! Around the office I've been discussing the possibility of shifting around and adding new activity modes to better group the stats together. I think it'll be a good change to make, though I don't know when we'll be able to get to it with our current backlog. It's definitely on my mind, however! Let's leave this open for when we can loop back to it.

**Comment by t3rr11** [comment]
Sounds good! Looking forward to seeing more endpoints, Doing some great things with the stats from those!

**Comment by vthornheart-bng** [comment]
Excellent!
